.github/workflows/trigger-e2e-tests.yml

```
@@ -0,0 +1,120 @@
+name: Trigger E2E Tests
+
+on:
+  pull_request:
+    types:
+      - ready_for_review
+  workflow_call:
+    inputs:
+      build-tag:
+        required: true
+        type: string
+
```

Let's set some flags for bash to make it more strict

```suggestion

defaults:
  run:
    shell: bash -euxo pipefail {0}
    
```